superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -411,7 +415,13 @@ class TabbedSqlEditors extends React.PureComponent {
         hideAdd={this.props.offline}
         onEdit={this.handleEdit}
         addIcon={
-          <Tooltip id="add-tab" placement="bottom" title="New tab (Ctrl + t)">
+          <Tooltip
+            id="add-tab"
+            placement="bottom"
+            title={
+              userOS === 'Windows' ? 'New tab (Ctrl + q)' : 'New tab (Ctrl + t)'
```

Comment:
Could use i18n here.

```suggestion
              userOS === 'Windows' ? t('New tab (Ctrl + q)') : t('New tab (Ctrl + t)')
```